CMakeLists.txt

```diff
@@ -32,12 +32,12 @@ SET(PROJECT_DESCRIPTION "${PROJECT_NAME_UCASE} - Free music production software"
 SET(PROJECT_COPYRIGHT   "2008-${PROJECT_YEAR} ${PROJECT_AUTHOR}")
 SET(VERSION_MAJOR       "1")
 SET(VERSION_MINOR       "1")
-SET(VERSION_PATCH       "91")
-#SET(VERSION_SUFFIX     "")
-SET(VERSION             "${VERSION_MAJOR}.${VERSION_MINOR}.${VERSION_PATCH}")
-IF(VERSION_SUFFIX)
-	SET(VERSION     "${VERSION}-${VERSION_SUFFIX}")
-ENDIF(VERSION_SUFFIX)
+SET(VERSION_BUILD       "91")
```

Comment:
If version is `1.2.0-rc2`, what should be done about [`FILEVERSION` numbers](https://github.com/LMMS/lmms/issues/2969#issuecomment-243505576)? Should they be dropped since a `FileVersion` string is also provided?

When version `1.2.0` is released, will you immediately tag `1.3.0-alpha`? The alpha tag could be configured not to be deployed.